app/views/login.ejs

```diff
@@ -51,9 +51,6 @@
           </div>
     </body>
 
-<script src="/js/admin-login.js"></script>
-<script src="/js/config.js"></script>
-<script src="https://www.gstatic.com/firebasejs/5.8.0/firebase.js"></script>
```

Comment:
These changes aren't in the other PR since the auth work isn't changing in that PR.  They're just getting removed here because they're still obsolete.  admin-login become login, config.js isn't used, and firebase is only required in the routes/login file.  These should have been removed with Jason's PR a while ago when moving auth to the backend.